src/index.js

```diff
@@ -0,0 +1,130 @@
+import 'regenerator-runtime/runtime';
+import axios from 'axios';
+
+const state = {
+    temperature: 40
+};
+
+const tempText = document.querySelector("#tempValue");
+const landscapeText = document.querySelector("#landscape");
+const cityText = document.querySelector("#cityNameInput");
+const skyText = document.querySelector("#sky");
+
+const changeSky = (sky) => {
+    if (sky === "sunny") {
+        skyText.innerHTML = "☁️ ☁️ ☁️ ☀️ ☁️ ☁️"
+    } else if (sky === "cloudy") {
+        skyText.innerHTML = "☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️"
+    } else if (sky === "rainy") {
+        skyText.innerHTML = "🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧"
+    } else if (sky === "snowy") {
+        skyText.innerHTML = "🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨"
+    }
+};
+
+const changeSkyValue = () => {
+    const skyValue = document.getElementById('skySelect').value;
+    changeSky(skyValue);
+};
+
+const increaseTemp = () => {
+    state.temperature += 1;
+    updateTempUi();
+};
+
+const updateTempUi = () => {
+    tempText.innerHTML = `${state.temperature}`;
+    if (state.temperature >= 80) {
+        landscapeText.innerHTML = "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂";
+        tempText.style.color = "red";
+    } else if (state.temperature <= 79 && state.temperature > 69) {
+        landscapeText.innerHTML = "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷";
+        tempText.style.color = "orange";
+    } else if (state.temperature <=69 && state.temperature > 59) {
+        landscapeText.innerHTML = "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃"; 
+        tempText.style.color = "yellow";       
+    } else if (state.temperature <=59 && state.temperature > 49) {
+        landscapeText.innerHTML = "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲";
+        tempText.style.color = "green"; 
+    } else if (state.temperature <=49) {
+        landscapeText.innerHTML = "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲";
+        tempText.style.color = "teal";
+    }  
+};
+
+const decreaseTemp = () => {
+    state.temperature -= 1;
+    updateTempUi();
+};
+
+const updateCityHeader = () => {
+    let currentCityInput = cityText.value;
+    const cityHeader = document.querySelector("#headerCityName");
+    cityHeader.innerHTML = `${currentCityInput}`;
+};
+
+const resetCityName = () => {
+    cityText.value = 'Seattle'
```

Comment:
Prefer to use a descriptively named variable to reference this value instead of a string literal. Something like:
```javascript 
const DEFAULT_CITY = 'Seattle';
cityText.value = DEFAULT_CITY;